src/driver/plugins.ts

```diff
@@ -0,0 +1,20 @@
+import type { Schema, Query } from 'mongoose';
+
+export function handle$vectorProjection(this: Query<unknown, unknown>, schema: Schema) {
```

Comment:
can we rename this to something like `handleVectorProjection` since this handles both `$vector` and `$vectorize` projections?